### ISSUE TITLE
Implementing Portfolio system

### ISSUE THREAD

**Issue by DaveBouman** [DigitalExcellence/dex-backend#315]
**Is your feature request related to a problem? Please describe.**
a portfoliosystem needs to be implemented, according to the [designs](https://github.com/DigitalExcellence/dex-backend/issues/268).

**Describe the solution you'd like**
The designs previously come up with need to be implemented. This entails new endpoints, additional or modified entities, new logic to handle user portfolios and portfolio items to be displayed in the front-end

**Additional context**
This can be done only after the research and designing has been done.

**Comment by niraymak** [comment]
We don't know yet how we want to show a portfolio in the frontend. We will give this issue low priority till we know more about it. We might be able to use the made implementation later.

**Comment by niraymak** [comment]
@RamonPeek did a research in what functionality is important for a portfolio system for our platform. This is done in issue. #388 

According to your research do you think we can use the branch that was created earlier?
https://github.com/DigitalExcellence/dex-backend/pull/347
